pkg/docker/util.go

```diff
@@ -155,3 +159,86 @@ func parseRepositoryTag(repos string) (string, string, string) {
 	}
 	return repos, "", ""
 }
+
+// PullImage pulls the Docker image specifies by name taking the pull policy
+// into the account.
+// TODO: The 'force' option will be removed
+func PullImage(name string, d Docker, policy api.PullPolicy, force bool) (*PullResult, error) {
```

Comment:
fyi this is here just to avoid writing the same switch everytime we want to pull the image (usage, incremental, build, etc)